packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -18,21 +18,43 @@ import { assert, performance } from "@fluidframework/common-utils";
  */
 export const latencyThreshold = 5000;
 
+interface IOpPerfTelemetryProperties {
+    /** Track how long op is sitting in inbound queue until it is processed */
+    durationInboundQueue: number | undefined;
+    /** Measure time outbound op is sitting in queue due to active batch */
+    durationOutboundQueue: number | undefined;
+    /** Time spent between DeltaManager's inbound "push" event until the DeltaManager "op" event */
+    durationInboundToProcessing: number | undefined;
+    /** Length of the DeltaManager's inbound queue at the time of the processing */
+    lenghtInboundQueue: number | undefined;
```

Comment:
Worth clarifying when we collect that metric (in what phase) - is it when op is added to inbound queue, or some other time?

Reply:
Added the reference to the Stage.